crates/aggkit-prover-config/tests/snapshots/validate_deserialize__empty_rpcs.snap

```diff
@@ -39,9 +38,10 @@ l1-rpc-endpoint = "http://anvil-mock-l1-rpc:8545/"
 [aggchain-proof-service.proposer-service.client]
 proposer-endpoint = "http://proposer-mock-rpc:3000/"
 sp1-cluster-endpoint = "https://rpc.production.succinct.xyz/"
+proving-timeout = 3600
 
-[aggchain-proof-service.proposer-service.client.proving-timeout]
-secs = 3600
+[aggchain-proof-service.proposer-service.client.request-timeout]
+secs = 600
```

Comment:
Could we change the config format for `request-timeout` too or do we need to keep this one for backwards compat?

Reply:
Sorry, I have missed that